### PR TITLE
Update Version

### PR DESCRIPTION
**Pull request by michaelpietzsch**
Updating Grafana version and user settings to correspond with the current container setup

### CODE REVIEW

File: grafana/grafana.libsonnet [hunk 1]
```diff
@@ -150,8 +150,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       deployment.mixin.spec.selector.withMatchLabels(podLabels) +
       deployment.mixin.spec.template.spec.withNodeSelector({ 'beta.kubernetes.io/os': 'linux' }) +
       deployment.mixin.spec.template.spec.withVolumes(volumes) +
-      deployment.mixin.spec.template.spec.securityContext.withRunAsNonRoot(true) +
-      deployment.mixin.spec.template.spec.securityContext.withRunAsUser(65534) +
+      deployment.mixin.spec.template.spec.securityContext.withRunAsUser(0) +
```
Comment (brancz) [inline grafana/grafana.libsonnet]: the default should not be running grafana as root, there is no reason to do this

Comment (michaelpietzsch) [reply grafana/grafana.libsonnet]: Well what about users that are migrating from an older version? Files created with a prior version won't have correct permissions. We could run as user 472. Would this be a viable solution for you?

Comment (brancz) [reply grafana/grafana.libsonnet]: There is nothing wrong with you setting permissions for your specific setup, but the default should be a totally unprivileged user.

Comment (michaelpietzsch) [reply grafana/grafana.libsonnet]: I unterstand and agree

**Comment by brancz** [comment]
Thanks!

**Pull request merged**